ydb/core/tx/datashard/change_exchange.h

```diff
@@ -27,30 +26,17 @@ struct TEvChangeExchange {
         EvActivateSenderAck,
 
         /// Local exchange (mostly using change's id)
-        // Enqueue for sending
-        EvEnqueueRecords,
-        // Request change record(s) by id
-        EvRequestRecords,
-        // Change record(s)
-        EvRecords,
-        // Remove change record(s) from local database
-        EvRemoveRecords,
-
         // Add new change sender
         EvAddSender,
```

Comment:
It looks dangerous to move message id's. Here i checked that all of them are local, but just as an idea for further development: maybe we should keep local message ids apart from remote one's?